docs/OIDC_provider_setup.md

```diff
@@ -0,0 +1,143 @@
+If you are using an enterprise plan, this is the callback url for it. http://localhost:8000/g/sso/oidc/Site_wide_SSO_Provider/login/callback/ 
+![image](https://user-images.githubusercontent.com/93269128/159430424-cfae2dbf-2346-4a0f-bfa6-dfda485433ca.png)
+7. Save changes after completing add callback urls.
+8. Disable grants to use organization function. 
+![image](https://user-images.githubusercontent.com/93269128/159425667-1da0d918-d729-41ed-9d8f-05af4e9929a2.png)
+9. Change the Organization setting to “Team members of organizations” and click “Save Changes“. 
+![image](https://user-images.githubusercontent.com/93269128/159425673-2c41e39d-3a46-41da-9428-f0127f60b8f3.png)
+
+
+### AUTH0 USER SETUP
+
+1. Go to ”User Management” and select the user to go to the user page. Click “Create User” to create a new user. 
+![image](https://user-images.githubusercontent.com/93269128/159425732-0a9052e1-b1a1-40ad-94af-4d82c3a88110.png)
+2. Fill in all of the fields and click “Create” to create a new user. 
+![image](https://user-images.githubusercontent.com/93269128/159425784-ef15759e-bb20-415d-b712-d65ac689dbe3.png)
+
+### AUTH0 ORGANIZATION SETUP
+
+1. Go to “Organizations” and click “Create Organization” to create a new organization. 	
+![image](https://user-images.githubusercontent.com/93269128/159425859-295b9e3b-1c59-4474-9697-07da4db63371.png)
+2. Name your organization and a name displayed.
+![image](https://user-images.githubusercontent.com/93269128/159425894-3a75a592-df49-459b-a68e-0794aab9dd9e.png)
+3. Please record this “Organization ID” because this ID has to use when create org sso id provider. 	
+![image](https://user-images.githubusercontent.com/93269128/159425928-da97e42b-cfb3-457e-ac29-34f7a67fff92.png)
+4. You can change these fields to adjust thein UI the login page. Save changes if you change anything.	
+![image](https://user-images.githubusercontent.com/93269128/159426034-79bf463c-7af6-4490-a7fe-449aab4b9fb0.png)
+5. Go to the “Member” session and click “Add Members” to add members to your organization. 
+![image](https://user-images.githubusercontent.com/93269128/159426219-9324b591-247c-4bd4-ad5b-2d2dab4188d0.png)	
+6. After selecting the user you want to add, click “Add Member” to add them to the organization.
+![image](https://user-images.githubusercontent.com/93269128/159426220-158eeab4-cb8c-4222-8c86-e087d8d64d0d.png)
+7. Go to "Connections" session and click "Enable Connections" to add connections to organization.
+![image](https://user-images.githubusercontent.com/93269128/200621154-9b8e4e11-ac1a-44f8-afc2-19b9f8a6ce49.png)
+8. Select "Username-Password-Authentication" and click "Enable Connection".
+![image](https://user-images.githubusercontent.com/93269128/200621448-e8314aff-21a8-4473-af76-5483c4695d6d.png)
+
+
+## KEYCLOAK SETUP
+
+### KEYCLOAK OIDC SERVER SETUP
+
+1. Login to the keycloak admin console. Default username is “admin” and passwords is “graphistry”. 
```

Comment:
Login to the keycloak admin console. Default username is “admin” and password is “graphistry”. **